modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/index.ts

```diff
@@ -351,4 +22,17 @@ export {
 	FDS_NESTED_FIELD_NAME_PARENT_SUFFIX,
 } from './constants';
 
+export {default as FDS_EVENT} from './utils/eventsDefinitions';
+
+export {
+	DisplayType,
+	IClientExtensionRenderer,
+	ICreationActionItem,
+	IFrontendDataSetProps,
+	IInfoPanelComponent,
+	IInternalRenderer,
+	IItemsActions,
+	IItemActionsData,
+} from './utils/types';
+
```

Comment:
@juanjofgliferay These exports were missing and causing TS errors in other modules. I ran `yarn node-scripts check:tsc --all` and there is only one error in `apps/object/object-dynamic-data-mapping-form-field-type`, that looks unrelated.

Reply:
In addition, I changed `TInfoPanelComponent` to `IInfoPanelComponent`, I didn't notice this in your PR. Reason is prefer [interfaces over types](https://ncjamieson.com/prefer-interfaces/).